### ISSUE TITLE
UG description is difficult to understand

### ISSUE THREAD

**Issue by adzikrafi** [adzikrafi/pe#7]
![Screen Shot 2023-04-14 at 16.55.11.png](https://raw.githubusercontent.com/adzikrafi/pe/main/files/d351a365-4594-45a7-a405-70588493d47a.png)
this part of UG is a bit wordy making it quite difficult to be understoob by the user

**Comment by nus-pe-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Though, there are other issues that pertain to this section that we have accepted, it is two sentences, so I don't think we will accept this under the bug report that it is "wordy".

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
